### ISSUE TITLE
Discussion on License Choice

### ISSUE THREAD

**Issue by hatefr** [UBC-MDS/lrasm#39]

**Comment by hatefr** [comment]
After having a through discussion on the choice of license, we decided `The MIT License` serves best our package. Through our search, we realized MIT is the most common open source license used for software development. MIT license is simple and straightforward, does not have many restrictions, and it allows the user to effectively use our package as an open source. This way, we also encourage the development from the people in the community and that will lead to faster growth and distribution of our package.  Based on MIT license, users are allowed to: 

- Use the code in commercial applications
- Developers can change/update the code
- Distribute copies of the code and any modifications
- Sublicense the code

We also compared the `MIT License` to other licenses such as `Apache 2.0` and `BSD License`. Although they share many common features such as they all require any reuse of the code to include the original copyright notice and a full-text copy of the license. However, `Apache License 2.0` is more strict about these terms. For example, this license states that anyone who significantly modifies the code must describe their changes. `BSD License` also uses a language which is less permissive. So, to conclude, given these restrictions, the `MIT License` remains the most favorite license for software development and that is what we use for our package in this repository. 

@AndyYang80 @gutermanyair  Please share here if you have any further thoughts.

**Comment by AndyYang80** [comment]
I agree, `MIT License` is a better choice as it is permissive, simple, flexible, and permits our package to be easily modified and published. I feel like more people are willing to contribute to our package in future if we distribute it under the `MIT License`. And since it is for educational purposes and we do not have proprietary materials in our package, I also suggest we proceed with the `MIT License`.

**Comment by gutermanyair** [comment]
I agree with all this!
